test/functional/rpc_psbt.py

```diff
@@ -78,17 +78,17 @@ def test_psbt_incomplete_after_invalid_modification(self):
         node = self.nodes[2]
         wallet = node.get_wallet_rpc(self.default_wallet_name)
         address = wallet.getnewaddress()
-        wallet.sendtoaddress(address=address, amount=1.0)
+        wallet.sendtoaddress(address=address, amount=Decimal("1.0"))
         self.generate(node, nblocks=1, sync_fun=lambda: self.sync_all(self.nodes[:2]))
 
         utxos = wallet.listunspent(addresses=[address])
-        psbt = wallet.createpsbt([{"txid": utxos[0]["txid"], "vout": utxos[0]["vout"]}], [{wallet.getnewaddress(): 0.9999}])
+        psbt = wallet.createpsbt([{"txid": utxos[0]["txid"], "vout": utxos[0]["vout"]}], [{wallet.getnewaddress(): Decimal("0.9999")}])
```

Comment:
we have [many](https://github.com/bitcoin/bitcoin/blob/master/test/functional/rpc_signrawtransactionwithkey.py#L81) [other](https://github.com/bitcoin/bitcoin/blob/master/test/functional/wallet_sendall.py#L291) [instances](https://github.com/bitcoin/bitcoin/blob/master/test/functional/wallet_bumpfee.py#L146) where we've kept the floating point format - can we set up a linter if that's important instead of fixing only a subset?

Reply:
I agree, but I'm not familiar with the Python linter capabilities.